src/pro/sky/java/course1/homework12/Author.java

```diff
@@ -0,0 +1,21 @@
+package pro.sky.java.course1.homework12;
+
+public class Author {
+    private String name_;
+    private String lastName_;
+
+    public Author(String name, String laseName) {
+        name_ = name;
+        lastName_ = laseName;
+    }
+
+    public String GetName(){
```

Comment:
```suggestion
    public String getName(){
```
названия методов пишем с маленькой буквы